extensions/kubeapi/projects/delete.go

```diff
@@ -0,0 +1,25 @@
+package projects
+
+import (
+	"context"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DeleteProject is a helper function that uses the dynamic client to delete a Project from a specific cluster.
+func DeleteProject(client *rancher.Client, project *v3.Project) error {
+	dynamicClient, err := client.GetDownStreamClusterClient(LocalCluster)
```

Comment:
Same as the above comment, should this be a local cluster?

Reply:
Project CRDs are available only on the local cluster and not on the downstream cluster.